jenkins-pipelines/longevity-2tb-4days-1Dis-2NonDis-Nemesises.jenkinsfile

```diff
@@ -9,5 +9,5 @@ longevityPipeline(
     test_name: 'longevity_test.LongevityTest.test_custom_time',
     test_config: 'test-cases/longevity/longevity-2TB-48h-authorization-and-tls-ssl-1dis-2nondis-nemesis.yaml',
 
-    timeout: [time: 3100, unit: 'MINUTES']
+    timeout: [time: 3120, unit: 'MINUTES']
```

Comment:
i think in many of longevity pipelines the timeout value is a bit longer that the yaml test duration.

Reply:
If it's a longevity pipeline the time can be removed @enaydanov, correct me if I'm wrong, but jenkins timeout should be calculated from 'test_duration' ?

Reply:
I also thought that but there are still many ones that use this.